config/database.yml

```diff
@@ -2,8 +2,8 @@ default: &default
   adapter: mysql2
   encoding: utf8
   username: root
-  password: 
-  host: 127.0.0.1
+  password:
+  host: <%= ENV['MYSQL_HOST'] || "127.0.0.1" %>
```

Comment:
A more rails way of doing this is you can use the ENV var of `ENV['DATABASE_URL']` to add whatever you want and isn't a code change anymore. See here: http://edgeguides.rubyonrails.org/configuring.html#connection-preference. This Env var takes precedence over the config file.